mne/io/edf/edf.py

```diff
@@ -613,7 +613,8 @@ def _read_edf_header(fname, exclude):
         for i, unit in enumerate(units):
             if i in exclude:
                 continue
-            if unit in ('uV', 'µV', 'µV'):
+            if unit in ('\u03BCV','\u00B5V','uV'):
+                #unicode symbols for microvolt
```

Comment:
```suggestion
            # allow both μ (greek mu) and µ (micro symbol) codepoints
            if unit in ('\u03BCV', '\u00B5V', 'uV'):
```

Reply:
FYI, our style checker demands spaces after commas, and also after `#` characters that start comments.